lib/active_merchant/billing/gateways/paystation.rb

```diff
@@ -24,6 +24,9 @@ class PaystationGateway < Gateway
 
       def initialize(options = {})
         requires!(options, :paystation_id, :gateway_id)
+		unless options.has_key?(:hmac_key)
```

Comment:
Fix whitespace? Not going to mark each place, but please review the whole patch for whitespace.